en/ros/ros2_comm.md

```diff
@@ -0,0 +1,511 @@
+# PX4-ROS2 bridge
+
+This section of the guide focusses on the bridge between PX4 and ROS 2, made available through the *microRTPS* bridge and resulting products to interface with ROS 2, *px4_ros_com* and *px4_msgs* ROS 2 packages.
```

Comment:
This sentence doesn't make sense to me.

What I think it says is that the "ROS2- bridge" is the RTPS Bridge + a set of ROS2 pages named *px4_ros_com* and *px4_msgs*. 
Its a bit confusing where the separation is, because aren't those packages built as part of the  RTPS bridge?

Anyway, a little more clarification, possibly in both docs, to make it clear the boundary/diferrences/purposes of the two bridges might help.

Reply:
PX4-RTPS bridge aka microRTPS bridge = FastRTPS bridge with a client in PX4 and an agent in the companion computer. The IDL files are generated on the PX4 firmware build processed before being used on the agent code generators (with fastrtpsgen).
PX4-ROS2 bridge aka `px4_ros_com` = it's the same bridge as the previous one, the difference being the templates for the agent code are difference, because the types used for ROS2 are different from the types used for the "raw" RTPS/DDS applications. `px4_msgs` is used to generated those data types and the IDL files consumed by the agent code generator in `px4_ros_com` (also with fastrtpsgen).

Reply:
I am not entirely sure how to translate this to the documents (also where). Can you help?